### ISSUE TITLE
Address contrast ratio problems with updated color palette

### ISSUE THREAD

**Issue by tannerdolby** [mdn/mdn-minimalist#717]
> These are different reds. The old one was #d0454c and the new one is #a30008. If the intention of this PR was just to refactor the variable names, then this is a "regression" (or something). _[original comment](https://github.com/mdn/yari/pull/4208#discussion_r669841694) by @peterbe_

After having a look at the code review in [#4208](https://github.com/mdn/yari/pull/4208) within `mdn/yari`. I did some investigating and the "new" red (`$mdn-red700`) used for  "bad" code examples has even worse contrast ratio than before.

Contrast Ratio: 3.8:1 (Fails mostly everything except WCAG AA for large text which is 3.5:1)
Foreground: #1b1b1b
Background: #d0454c

<img width="913" alt="Screen Shot 2021-07-27 at 9 58 17 AM" src="https://user-images.githubusercontent.com/48612525/127196440-0cb24852-2aac-47b0-bf16-b70863f6f1c6.png">

to an even worse contrast ratio with 

Contrast Ratio: 2.1:1 (Fails WCAG AA and AAA)
Foreground: #1b1b1b
Background: #A30008

<img width="1031" alt="Screen Shot 2021-07-27 at 9 58 32 AM" src="https://user-images.githubusercontent.com/48612525/127196463-40c21868-317e-469e-bfbb-7cb4c547f4c8.png">

**Comment by tannerdolby** [comment]
This should be addressed ASAP

cc @schalkneethling
